Joss paper (#273)
insightsengineering/rbmi: misc/joss_manuscript/paper.md

--- 
title: 'rbmi: A R package for standard and reference-based multiple imputation methods'
author: 
 - Craig Gower-Page^[Data & Statistical Sciences, Pharma Development, Roche, Welwyn Garden City, UK] 
 - Alessandro Noci^[Data & Statistical Sciences, Pharma Development, Roche, Basel, Switzerland] 
 - Marcel Wolbers$^{\dagger}$
tags:
  - R
  - Biostatistics
  - Clinical Trials
  - Estimands
  - Missing Data
  - Multiple Imputation
  - Reference-based Methods
output: 
  bookdown::pdf_book:
    toc: false
    citation_package: natbib
date: \today
bibliography: references.bib
link-citations: true
---

# Summary {-}

> Many randomized controlled clinical trials compare a continuous outcome variable which is assessed longitudinally at scheduled follow-up visits between the subjects assigned to the intervention treatment and those assigned to the control group. 

Author: ```suggestion
randomized controlled clinical trials compare a continuous outcome variable which is assessed longitudinally at scheduled follow-up visits between the subjects assigned to the intervention treatment and those assigned to the control group. 
```

The `draws()` function has 3 input arguments: 

- `data`: The primary longitudinal `data.frame` containing the outcome variable and all covariates. The inclusion of time-varying covariates is also possible.
- `data_ice`: A `data.frame` which specifies the first visit affected by an ICE and the imputation strategy for handling missing outcome data after the ICE.
- `method`: The selected statistical approaches which may be defined by creating a `method` object using the functions:
    - `method_bayes()` for MI based on Bayesian posterior parameter draws from MCMC sampling and inference based on Rubin's rules [@CarpenterEtAl2013].
    - `method_approxbayes()`: as for `method_bayes()` except that  approximate Bayesian posterior draws are obtained via bootstrapping and maximum likelihood estimation (@LittleRubin1992[Section 10.2.3, part 6]).
    - `method_condmean()` for conditional mean imputation based on maximum likelihood estimation. Inference is based on re-sampling techniques (bootstrap or jackknife) as described in @Wolbers2021.
    - `method_bmlmi()` for bootstrapped maximum likelihood MI as described in @vanHippelBartlett2021.

> In addition to detailed help files for all functions, the package contains three vignettes: a `quickstart` vignette which describes the basic functionality, an `advanced` vignette which describes some of the advanced features, and a `stat_specs` vignette which describes the statistical methodology in detail. 

Reviewer: ```suggestion
In addition to the detailed help files for all functions, the package contains three vignettes: a `quickstart` vignette which describes the basic functionality, an `advanced` vignette which describes some of the advanced features, and a `stat_specs` vignette which describes the statistical methodology in detail. 
```

> # Availability and validation {-}
> 
> `rbmi` is developed open source on https://github.com/insightsengineering/rbmi and major releases will also be uploaded to [CRAN](https://cran.r-project.org/).
> Source code can only be promoted to the master branch after review by a second programmer. Unit tests which define and document the expected input and output of each function have been implemented to ensure that the package performs as expected. To date, `rbmi` has been used in two simulation studies reported in @Wolbers2021 and @Noci2021.
> 

Reviewer: ```suggestion
# Development & Implementation {-}

`rbmi` is developed open source on https://github.com/insightsengineering/rbmi and major releases will also be uploaded to [CRAN](https://cran.r-project.org/).
All production code is required to have been reviewed by an independent programmer as as pass a substancial suite of unit tests which both define and document the expected input and output of each function. These practices ensure that the package is of the highest standard and performs as expected. Additionally comparisons are made to similar software (namely the so-called "five macros" [@FiveMacros] SAS implementation) to ensure consistency of results as well as to simulated datasets with known values.
To date, `rbmi` has been used in two simulation studies reported in @Wolbers2021 and @Noci2021.

```
